silo/tests/test_views.py

```diff
@@ -225,9 +227,23 @@ def test_export_csv(self):
 
 class SiloViewsTest(TestCase):
     def setUp(self):
+        factories.ReadType(read_type='CustomForm')
         self.org = factories.Organization()
         self.tola_user = factories.TolaUser(organization=self.org)
         self.factory = APIRequestFactory()
+        self.silo_id = None
+
+    def tearDown(self):
```

Comment:
I saw that in two classes we are removing the values. Can't we do that better in `MongoTestCase` and inherit from there?